resources/assets/css/front/components/_embed.scss

```diff
@@ -0,0 +1,13 @@
+.embed {
```

Comment:
Aren't there utilities in tailwind that enabled us to write this less verbose?

Reply:
It can be done with tailwind but it means more html replacement in the `getTextAttribute()` method.